fastestimator/layers/pggan_layers.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import tensorflow as tf
```

Comment:
rather than a single 'pggan_layers' file, the layers should be broken out individually. see existing examples. also then need to be added to the init.py

Reply:
all of those layers are very specific to PGGAN, so I guess we should be alright